problems/removeNumbersAtOddIndices.js

```diff
@@ -2,8 +2,11 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
-
+function removeNumbersAtOddIndices(arr) {
+let output = arr.filter((el) =>{
```

Comment:
You're checkin the elements instead of the indicies. Remember the callback on array methods can also take the index as a second argument.